src/parser.rs

```diff
@@ -1726,6 +1729,19 @@ impl<'a> Parser<'a> {
         Ok(ObjectName(idents))
     }
 
+    /// Parse identifiers
+    pub fn parse_identifiers(&mut self) -> Result<Vec<Ident>, ParserError> {
+        let mut idents = vec![];
+        loop {
+            match self.next_token() {
+                Token::Word(w) => idents.push(w.to_ident()),
+                Token::EOF => break,
+                _ => {}
+            }
+        }
+        Ok(idents)
+    }
```

Comment:
Is `SHOW` (without any variable) valid SQL? This code would return  `Ok(vec![])` for the SQL code `SHOW`, correct?

Reply:
yes